lms/static/js/learner_dashboard/models/course_card_model.js

```diff
@@ -9,17 +9,34 @@
         function (Backbone) {
         return Backbone.Model.extend({
             initialize: function(data) {
+                this.activeRunMode = null;
                 if (data){
                     this.context = data;
                     //we should populate our model by looking at the run_modes
                     if (data.run_modes.length > 0){
-                        //We only have 1 run mode for this program
-                        this.setActiveRunMode(data.run_modes[0]);
+                        if(data.run_modes.length === 1){
+                            this.setActiveRunMode(data.run_modes[0]);
+                        }
+                        else{
+                            this.setActiveRunMode(
+                                this.getMostRecentRunMode(data.run_modes));
+                        }
                     }
                 }
             },
 
+            getMostRecentRunMode: function(runModes){
+                //Need to expand this function to retrieve
+                //the most relevant runmode for the course
+                return runModes[0];
+            },
+
+            getActiveRunMode: function(){
```

Comment:
I'm not convinced we need this, any data should be accessible directly from the model.